src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -605,8 +605,10 @@ def _to_series_list(self, index: pd.Index) -> list[pd.Series]:
                 pd.Series(index.get_level_values(level))
                 for level in range(index.nlevels)
             ]
-        elif isinstance(index, pd.Index):
+        elif isinstance(index, (pd.Index, pandas.Index)):
             return [pd.Series(index)]
+        else:
+            raise TypeError("Index must be pd.Index or pd.MultiIndex object")
```

Comment:
Based on the condition above, shouldn't shit say Index must be Modin Index or pandas Index?

Reply:
+1 should not have change here.

Reply:
Ok should I just remove the else case entirely? I only put it there because it was silently failing and returning None when given modin Index object, because it was not expecting it. Is that functionality when given a bad type ok?